package.json

```diff
@@ -13,7 +13,8 @@
     "react-dev": "webpack --watch",
     "seed-database": "node database/seed.js",
     "data-gen": "node database/data_generation.js",
-    "neo4j-gen": "node database/data_generation_neo4j.js"
+    "neo4j-gen": "node database/data_generation_neo4j.js",
+    "k6": "node k6_http_get.js"
```

Comment:
will you be adding a file & script command for stress testing post reqs as well?